client/inbox-panel/index.js

```diff
@@ -164,6 +164,7 @@ const INBOX_QUERY = {
 		'image',
 		'is_deleted',
 		'is_read',
+		'locale',
```

Comment:
I was going to suffix locale value with `_experiment` as the issue description suggested, but the value is already available from the backend.

Reply:
Nice! I didn't know we can just add parameter in our query if it was already defined in backend.